### ISSUE TITLE
BibTex import does not work for Scopus exports

### ISSUE THREAD

**Issue by mietcls** [ugent-library/biblio-backoffice#801]
# Bug

Some files work, some files do not:
https://ugentbe-my.sharepoint.com/:f:/g/personal/miet_claes_ugent_be/ElBbCmVO64tIuGUZXD7c2EQBW5Afpjnt9h7Xq3Gu01J3_Q?e=HUsEnx

Neither `.txt`, `.bib` nor `.csv` extensions work.

Hunch why it does not work:
- scopus restructured their files?
- we only accept `.txt`, however these also do not work any longer

## Steps to reproduce:
1. Go to scopus https://www.scopus.com/search/form.uri?display=basic#basic, and [search for a selection](https://www.scopus.com/results/results.uri?sort=plf-f&src=s&st1=UGent&sid=74f3247161331f7857b4994d284f410d&sot=b&sdt=b&sl=20&s=TITLE-ABS-KEY%28UGent%29&origin=searchbasic&editSaveSearch=&yearFrom=Before+1960&yearTo=Present&sessionSearchId=74f3247161331f7857b4994d284f410d&limit=10)
2. Click "Export" and pick any you like
3. Try uploading them in Biblio

## Screenshots

Failed import:
![Image](https://user-images.githubusercontent.com/1572885/201054128-820c0c59-e8ae-414b-be74-a68ef175e794.png)

## Steps to reproduce

- Go to back office
- Upload bibtex file (see attachments)
- See error

## Expected behaviour
As a researcher, I can export files from Scopus in Bibtex, and import it without adapting the files.

**Comment by nics** [comment]
there are 3 problems with these files:
- unescaped @ in email addresses
- trailing comma after last field
- field names containing spaces

**Comment by mietcls** [comment]
Todo Miet: Add new examples

**Comment by nicolasfranck** [comment]
@mietcls are there more failing examples?

**Comment by mietcls** [comment]
Updated with examples. You can try exports from Scopus as well. We use this guidance: https://onderzoektips.ugent.be/en/tips/00002068/ (a bit outdated, but the instructions remain valid)

**Comment by nicolasfranck** [comment]
@mietcls wouldn't it be more prudent to report these problems to scopus? I guess we are not the only ones who cannot import these files. "Fixing" these problems requires to rewrite the module, or to "fix" the incoming (and so probably hypercorrectly introduce other problems).

**Comment by mietcls** [comment]
@nicolasfranck can we first figure out what the issue is? Perhaps it's something on our side that changed. This issue dates from our launch in 2022, a year ago.

**Comment by mietcls** [comment]
TLDR from a bunch of conversations:
- BibTeX is wonky
- can we use RIS (less informative than BibTeX) or CSV (risky expectation management) instead?
- We need to figure out how many people use bibtex

**Comment by mietcls** [comment]
Closing this one as it's outdated
